### PR TITLE
ASSJAM - Disables zipgun, adds a slamgun

### PR DESCRIPTION
**Pull request by polivilas**
## About the PR
ASSJAM only for now.
Adds a slamgun (simple 12gauge shotgun made from two pipes), removes zipgun.
Also included in PR: Fabricable 12gauge shotgun ammo, 'nailshot': Very low damage versus armored targets, while absolutely devastating against unarmored targets.
As a balancing factor, you need to extract shells after firing manually.
Video: https://puu.sh/FwftF/0135d88b3a.mp4

## Why's this needed?
Adds a important 'stepping stone' to allow better kinetic weapon variation without having to fear zipgun as a un-balancer. Moreover, makeshift weapons would fit very well in gang gamemode.

## Bounty
I will pay out 100k spacebux for better sprites - contact 2027#2027 on discord for more details.

## Changelog

```
2027:
* Assjam - Adds a slamgun, removes zipgun.
```

### CODE REVIEW

**Comment by Tarmunora** [comment]
Potential balance concern for barman printing off 12gaNailShot and having similar stopping power to spes-12 against unarmored targets at little to no cost.

File: code/modules/projectiles/special.dm [hunk 1]
```diff
@@ -191,6 +191,15 @@
 		FC.dissipation_ticker = rand(0,dissipation_variance)
 		FC.launch()
 
+/datum/projectile/special/spreader/buckshot_burst/nails
```
Comment (Tarmunora) [inline code/modules/projectiles/special.dm:194]: Consider adding proper projectile stats (power, damage type) for pointblanking

Comment (polivilas) [reply code/modules/projectiles/special.dm:194]: Should be fixed by 757ac52

**Comment by polivilas** [comment]
> 
> 
> Potential balance concern for barman printing off 12gaNailShot and having similar stopping power to spes-12 against unarmored targets at little to no cost.
Armor is quite easy to get, in basic form (engineering spacesuits, for example), and extreme loudness of ammo should very quickly clue everyone in into what's happening. 
I do understand the concern tho, and that's one of the reasons why I want to push it to assjam first.

File: code/modules/projectiles/special.dm (hunk 1 above)
Comment (Tarmunora) [reply code/modules/projectiles/special.dm:194]: Intentional that the pointblank is outright stronger than regular firing at 'point blank range'? (note that general behavior for pointblanking is to be the same as a direct hit)

**Comment by geneva-breaker** [comment]
haha pipe gun go bang bang

File: code/modules/projectiles/special.dm (hunk 1 above)
Comment (polivilas) [reply code/modules/projectiles/special.dm:194]: Pointblanking doesn't give any bleed, so I buffed on point-blank range to compensate for that :v

**Comment by Tenryumi** [comment]
Tbh, I love this. Scrappy makeshift guns and ammo are rad and so is this PR

**Pull request merged**